commitlint.config.ts

```diff
@@ -356,7 +356,11 @@ module.exports = {
         'too-many-spaces': [RuleStatus.Error, 'always'],
         'commit-hash-alone': [RuleStatus.Error, 'always'],
         'title-uppercase': [RuleStatus.Error, 'always'],
+        'proper-revert-message': [RuleStatus.Error, 'always'],
     },
+    // Commitlint automatically ignores some kinds of commits like Revert commit messages. 
```

Comment:
2 last things to do in this PR:
- (Nit) Add an EOL before this comment above, to visually separate it better from the rule configuration list.
- Remove the TODO comment related to this PR, at line 363.